### ISSUE TITLE
Cert Resource constantly updates

### ISSUE THREAD

**Issue by troywilson** [terraform-providers/terraform-provider-digitalocean#146]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

0.11.8

### Affected Resource(s)
Please list the resources as a list, for example:
- digitalocean_certificate

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl

resource "tls_private_key" "primary-cert-private_key" {
  algorithm = "RSA"
}

resource "tls_cert_request" "primary" {
  key_algorithm   = "RSA"
  private_key_pem = "${tls_private_key.primary-cert-private_key.private_key_pem}"
  dns_names       = ["${var.primary_dns_name}", "*.${var.primary_dns_name}"]

  subject {
    common_name = "${var.primary_dns_name}"
  }
}

resource "acme_certificate" "primary" {
  account_key_pem         = "${tls_private_key.reg_private_key.private_key_pem}"
  certificate_request_pem = "${tls_cert_request.primary.cert_request_pem}"
  depends_on              = ["aws_route53_record.primary-caa-wildcard"]
  min_days_remaining      = 31

  dns_challenge {
    provider = "route53"
  }
}

resource "digitalocean_certificate" "primary" {
  name              = "${var.primary_dns_name}"
  private_key       = "${tls_private_key.primary-cert-private_key.private_key_pem}"
  leaf_certificate  = "${acme_certificate.primary.certificate_pem}"
  certificate_chain = "${acme_certificate.primary.issuer_pem}"
}
```

### Expected Behavior
The cert is installed once and only re-installed when the underlining acme cert is changed.

### Actual Behavior
The cert is deleted and re-installed into DO every time. It was working fine before the 1.0 upgrade. Sample output from apply:

-/+ digitalocean_certificate.primary (new resource required)
      id:                 "xxxxxxxxxxxxxxxxxxxxxxxxx" => \<computed\> (forces new resource)
      certificate_chain:  "DATA CLIPPED"
      domains.#:          "2" => "0" (forces new resource)
      domains.xxxxxxx: "domain1.com" => "" (forces new resource)
      domains.xxxxxxx:  "*.domain1.com" => "" (forces new resource)
      leaf_certificate:   "DATA CLIPPED"
      name:               "domain1.com" => "domain1.com"
      not_after:          "2018-12-30T22:13:49Z" => \<computed\>
      private_key:        \<sensitive\> => \<sensitive\> (attribute changed)
      sha1_fingerprint:   "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx" => \<computed\>
      state:              "verified" => \<computed\>
      type:               "custom" => "custom"

Plan: 1 to add, 0 to change, 1 to destroy.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by andrewsomething** [comment]
Thank you for reporting this issue. I believe we have determined the cause and have a fix in place. I'll update here once we've cut a new release.

**Comment by troywilson** [comment]
Thanks for the quick turn around :)

**Comment by andrewsomething** [comment]
This fix was released as version 1.0.2. You can upgrade with:

    terraform init -upgrade

Thanks again for the report!
